system-tests/project-fixtures/react/cypress/cypress-webpack-dev-server-function.config.ts

```diff
@@ -0,0 +1,17 @@
+import { defineConfig } from 'cypress'
```

Comment:
Can the original file this was copied from be removed now?  It looks like it is no longer used by anything.